CODE_OF_CONDUCT.md

```diff
@@ -23,14 +23,13 @@ Examples of behavior that contributes to creating a positive environment include
 
 Examples of **unacceptable** behavior by participants include:
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
- advances.
+* The use of sexualized language or imagery and unwelcome sexual attention or advances.
```

Comment:
These seem to be just whitespace changes - are they helpful?

Reply:
Yes, I just noticed few whitespaces so I removed it.